### PR TITLE
Add service config for rabbitmq server, with more file handles

### PR DESCRIPTION
**Pull request by carlskeide**
Add recommended ulimit allotment to rabbitmq init settings

### CODE REVIEW

File: blues/rabbitmq.py [hunk 1]
```diff
@@ -41,6 +41,12 @@ def setup():
     else:
         install_stable()
 
+    blueprint.upload('default/rabbitmq-server',
+                     '/etc/default/rabbitmq-server',
+                     context={
+                        'ulimit': blueprint.get('rabbitmq.ulimit', '102400')
```
Comment (zalphi) [inline blues/rabbitmq.py]: i´m unfamiliar with the syntax. My suggestion `blueprint.get('ulimit', '102400')`

Comment (carlskeide) [inline blues/rabbitmq.py]: As was I, fixing this

**Comment by carlskeide** [comment]
Ok nu?

**Comment by zalphi** [comment]
👍

**Pull request merged**